### PR TITLE
[KO] Update ko.md of People/Global_Moderation_Team, People/Quality_Assurance_Team.

### PR DESCRIPTION
**Pull request by libneko**
### Description

Re-translation of those pages for fixing many things.

### Status

Finished

### Changelog

- Update the list of GMT, QAT with translation.
- Improve some sentences.

### CODE REVIEW

File: wiki/People/Global_Moderation_Team/ko.md [hunk 1]
```diff
@@ -1,62 +1,71 @@
...
 ---------------
 
-**글로벌 모더레이터** 가 되려면 상당히 어렵고 복잡한 과정을 거쳐야합니다. 제2언어를 구사할 수 있는 사람들이 종종 GMT 구성원이 될 가능성이 있는 사람들로 지목되며, 이후에 팀 구성원으로서의 잠재력에 대해 팀의 논의를 거쳐 선출됩니다.
+**글로벌 모더레이터** 가 되려면 상당히 어렵고 복잡한 과정을 거쳐야 합니다. 제2언어를 구사할 수 있는 사람들이 종종 GMT 구성원이 될 가능성이 있는 사람들로 지목되며, 이후에 팀 구성원으로서의 잠재력에 대해 논의 후 선출됩니다.
 
 역할과 책임
 -----------
 
-글로벌 모더레이터는 채팅 채널과 포럼의 쾌적한 운용을 책임지고 커뮤니티의 대화에 신경을 써야합니다. 만약 여러분이 채팅(사일, 괴롭힘 등)이나 포럼(스팸)에 대해서(이것에 한정된 것은 아님) 게임의 중재 및 도움 측면에서 뭔가 질문이 있다면 이들과 연락을 취할 수 있습니다.
+글로벌 모더레이터는 채팅 채널과 포럼의 쾌적한 운용을 책임지고 커뮤니티의 대화에 신경을 써야합니다. 만약 여러분이 채팅(사일, 괴롭힘 등)이나 포럼(스팸)에 대해서 (이것에 한정된 것은 아님) 채팅의 중재 및 게임적인 도움이 필요하다면 이들에게 연락해주세요.
```
Comment (Alex4386) [inline wiki/People/Global_Moderation_Team/ko.md]: It might be better to say **게임에 관련되어** instead of **게임적인**

Comment (libneko) [reply wiki/People/Global_Moderation_Team/ko.md]: Changed to "게임과 관련된".

**Comment by sr229** [comment]
please be reminded you have merge conflicts @libneko, please resolve them.

**Comment by libneko** [comment]
Closed for now.

**Pull request closed without merging**